assets/src/styles/blocks/core-overrides/_media-text.scss

```diff
@@ -11,6 +11,11 @@
     cursor: pointer;
   }
 
+  &.is-style-parallax .wp-block-media-text__media {
+    overflow: hidden;
+    z-index: -1;
+  }
```

Comment:
This makes the image disappear in the editor for me, do you have this too ? (the z-index specifically, on firefox)

Reply:
Good catch! I think that [this commit](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/931/commits/511de74b689fb7d615262a01aa0fe61d3de0d7a4) solves the problem 🙂